### PR TITLE
Add KB about Instruction: How To Migrate Longhorn Chart Installed In Old Rancher UI To The Chart In New Rancher UI

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#3714

### CODE REVIEW

**Comment by PhanLe1010** [comment]
Note: need to link to the migration script after [the PR](https://github.com/longhorn/longhorn/pull/4095) gets megered

**Comment by innobead** [comment]
@PhanLe1010 Please fix the DCO error.

**Comment by PhanLe1010** [comment]
Thanks @innobead . Fixed

File: content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    ```
+4. Go to Rancher UI -> Go to the downstream cluster -> App&Marketplace -> Charts -> Find and select Longhorn chart
+   -> Select the chart version corresponding to the correct Longhorn version
+   -> Install the chart with the correct helm values.
+   Note that all this detailed information is printed out at the end output of the above script.
+5. Verify that the migrated chart (in Rancher UI -> Downstream Cluster -> App & Marketplace -> Installed Apps) is working ok. E.g., check all Longhorn resources inside the namespace `longhorn-system`,
+   try to scale up and down workload to attach/detach Longhorn volumes, ...
+6. At this point the chart is migrated. However, you may notice that the old installation is still visible inside the legacy section:
+   {{< figure src="/img/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/longhorn-installation-in-legacy-ui.png" >}}
+7. To remove it so you don't accidentally use it in the future, you can run `longhorn_rancher_chart_migration.sh` script with the flag `--type=cleanup`.
```
Comment (shuo-wu) [inline content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md:172]: `To remove the legacy icon without affecting Longhorn......`

File: content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+## Prerequisites
+1. Longhorn v1.2.x and v1.3.x versions
+2. Rancher >= v2.6.3 versions
+3. The command [kubectl get-all](https://github.com/corneliusweig/ketall#installation) is available on your local machine
+
+## Steps
+
+1. Go to Longhorn UI -> Settings -> General -> Change Concurrent Automatic Engine Upgrade Per Node Limit setting to 0 to disable auto engine upgrade.
```
Comment (shuo-wu) [inline content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md:46]: NIT: Indicate how to use `kubectl` to update this.

File: content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+At this moment, there are 2 different Longhorn charts. One is in the App Catalog UI (you already installed Longhorn using this chart). The other one in the App & Marketplace (you have not installed Longhorn using this chart) as below:
+
+{{< figure src="/img/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui/longhorn-chart-in-new-ui.png" >}}
+
+Because the App Catalog UI is being deprecated, you may want to migrate the existing Longhorn installation to the Longhorn chart in the App & Marketplace UI.
+The reason is Longhorn is planing to stop releasing new versions in the App Catalog UI in the future.
+
+
+## Prerequisites
```
Comment (shuo-wu) [inline content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md:39]: Remember to provide the script download URL

**Review by shuo-wu — COMMENTED** [review]
In general LGTM

File: content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md (hunk 2 above)
Comment (PhanLe1010) [reply content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md:46]: Thank you. Fixed

File: content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md (hunk 1 above)
Comment (PhanLe1010) [reply content/kb/how-to-migrate-longhorn-chart-installed-in-old-rancher-ui-to-the-chart-in-new-rancher-ui.md]: Thank you. Fixed

**Review by innobead — APPROVED** [review]

**Pull request merged**